src/util/recalls.ts

```diff
@@ -0,0 +1,110 @@
+import { RecallsSchema } from '@dvsa/cvs-type-definitions/types/v1/recalls';
```

Comment:
Thinking about this, and I am not precious about it but in keeping with the structure of the existing repo...

Arguably the `getMotRecallsByVin` and `getBearerToken` methods could reside in some sort of `api` service (in the service directory), with a `processRecalls` (in the processors directory) subsequently utilising the `filterMotRecalls` method in utils.

It would clean up the structure a bit.

Reply:
Also appreciate we have to balance more work vs getting this done.

Reply:
The processors are used liberally and not consistently anyway. They more cover full processes. such as swapping a vin, or VRM. This would be overkill and not needing one imo.